Cookbook/Technical-Documents/CRP-Bot.md

```diff
@@ -0,0 +1,101 @@
+
+Again, this might seem surprising at first, but this is expected and on purpose.
+
+### Impact on "Fix versions" vs feature flags
+
+Another impact of that process is that **even features that are gated by a local feature switch** (hardcoded as being turned off in code) and end up still being disabled when we release the app **will still be flagged with the released version in their "Fix Version"**.
+
+This is because – similar to what was described above with the example of unwanted side effects of a US feature on the UK app – the implementation of a feature that is theoretically supposed to be disabled by a feature flag can accidentally introduce bugs or have some of the code not protected behind the feature flag like it should have been. So since the code is present in the final app anyway – even if it's supposed to be disabled – the corresponding JIRA ticket will still have the version set in its "Fix Version".
+
+This isn't really an issue in the end since usually we'll end up having a separate ticket to enable the feature switch and the feature in the future. So that new JIRA ticket about turning the feature on will have its own "Fix Version" telling us when it was enabled.
+
+Still, this is a behavior to be aware of to understand the real meaning we put behind that "Fix Version" field (i.e. "the code implementing this feature ended up in the codebase shipped on the store for this version, even if supposedly disabled").
+
+### "Fix versions" and planning versions ahead
+
+Some teams might want to use JIRA versions to plan ahead, at the beginning of a sprint, which tickets we hope to be part of which version.
+
+But since there's no guarantee that those tickets will make it in time in the planned release when the corresponding release branch is cut, it would lead to inconsistencies to use the Fix Version and JIRA releases named `"<platform> <app> <version>"` to mark those planned tickets before the release is cut; because if those planned tickets end up not making the cut, you'd still have manually marked them with a Fix Version that is supposed to reflect what was _actually_ shipped as part of the release.
+
+> Note that for each board, if a Jira version with the exact same name the bots is planning to use ("<Platform> <Flavor> <Version>" e.g. "iOS Ascension 1.1.0") already exists in the Jira board by the time the CRP process is run (= at the time the release branch is cut and the release process starts every second Friday), then the bot will re-use that existing Jira version instead of creating a separate one (Jira doesn't allow two versions to have the same name anyway)
+
+So for teams which want to use JIRA releases and some field to mark tickets _planned_ for that release ahead of time (e.g. at the beginning of the sprint), to avoid inconsistencies between what the bot uses for "tickets actually shipped" and what you use for "tickets hoping to make the cut", we suggest one of those two solutions:
```

Comment:
```suggestion
So for teams which want to use JIRA releases and some field to mark tickets _planned_ for that release ahead of time (e.g. at the beginning of the sprint), we suggest one of these two solutions in order to hopefully avoid inconsistencies:
```

Reply:
I also removed some wording because it is redundant, considering the context of this section